ui/src/views/Clients/ViewClient/index.jsx

```diff
@@ -165,8 +168,26 @@ export default class ViewClient extends Component {
     this.setState({ accessToken: null });
   };
 
+  // Only localhost callback_url's are whitelisted.
+  // This tool is not intended for other uses
+  // than setting up credentials via `taskcluster signin`,
+  // which uses this URL format.
+  isWhitelistedCallback = callbackUrl =>
```

Comment:
I think we're trying to get away from calling things white/black list but otherwise this makes sense.